src/types/posters.ts

```diff
@@ -3,4 +3,5 @@ export type Poster = {
   categories: string[];
   introVideoUrl: string;
   authorName: string;
+  thumbnailUrl: string;
```

Comment:
Is this guaranteed to always be defined on every poster? My guess is probably not, so we should probably mark it as 'optional', and then handle that potentially missing state appropriately in the code where needed:
```suggestion
  thumbnailUrl?: string;
```